R/compareScenarios.R

```diff
@@ -36,6 +36,8 @@ compareScenarios <- function(mif, hist,
                              y_bar=c(2010,2030,2050,2100),
                              reg=NULL, mainReg="GLO", fileName="CompareScenarios.pdf",
                              sr15marker_RCP=NULL) {
+  message("compareScenarios is outdated and may break - use compareScenarios2 instead!")
+
```

Comment:
The standard way to deprecate a function is to use
`.Deprecated("compareScenarios2")`
That will generate the correct messages / warnings.

Is it useful to keep `compareScenarios`? Or would it be less confusing to remove it outright?

Reply:
I changed that to `.Deprecated`. If somebody wants to remove it, that is ok, but I want just avoid that people use it and get confused as today in the REMIND mattermost channel.